src/main/java/service/UserMapper.java

```diff
@@ -0,0 +1,14 @@
+package service;
+
+import dto.UserResponseDto;
+import model.User;
+
+public class UserMapper {
+    public static UserResponseDto toDto(User user) {
```

Comment:
why do we make this method static?
let's use annotation @ Component above UserMapper and remove static from the method